src/integrationTest/java/uk/gov/hmcts/reform/pip/data/management/service/schemavalidation/nonstrategic/UtIacJudicialReviewsBirminghamDailyHearingListTest.java

```diff
@@ -78,13 +78,13 @@ void testValidateWithErrorsWhenHearingTimeMissingInList() throws IOException {
     }
 
     @Test
-    void testValidateWithErrorsWhenApplicantMissingInList() throws IOException {
```

Comment:
Missing integration tests for ValidationServiceTest and PublicationManagementTest